spec/helpers.lua

```diff
@@ -1388,13 +1388,27 @@ local function wait_until(f, timeout, step)
 end
 
 
----wait for some timer
----@param timer_name_pattern string
----@param plain boolean
----@param mode string all-finish | all-running | any-finish | any-running | worker-wide-all-finish
----@param timeout? number optional, maximum time to wait (default = 2)
----@param admin_client_timeout? number optional, to override the default timeout setting
----@param forced_admin_port? number optional, to override the default port of admin API
+---Wait for some timers, throws an error on timeout.
+--- 
+--- NOTE: this is a regular Lua function, not a Luassert assertion.
+---@function wait_timer
+---@param timer_name_pattern string the call will apply to all timers matching this string
+---@param plain boolean if truthy, the `timer_name_pattern` will be matched plain, so without pattern matching
+---@param mode string one of: "all-finish", "all-running", "any-finish", "any-running", or "worker-wide-all-finish"
+--- 
+---any-finish: At least one of the timers that were matched finished
+--- 
+---all-finish: All timers that were matched finished
+--- 
+---any-running: At least one of the timers that were matched is running
+--- 
+---all-running: All timers that were matched are running
+--- 
+---worker-wide-all-finish: All the timers in the worker that were matched finished
+---@param[opt=2] timeout number optional, maximum time to wait
+---@param[opt] admin_client_timeout number optional, to override the default timeout setting
+---@param[opt] forced_admin_port number optional, to override the default port of admin API
+---@usage helpers.wait_timer("rate-limiting", true, "all-finish", 10)
```

Comment:
can we please make this consistent with the rest of the file? triple-dash + space on the first line, double-dash + space on the following lines